package.json

```diff
@@ -37,6 +38,7 @@
   "dependencies": {
     "bootstrap": "^4.5.0",
     "clipboard": "^2.0.6",
+    "extension-build-webpack-plugin": "^1.1.7",
```

Comment:
I don't see it being used in code. What is the purpose of this?

Reply:
I didn`t noticed this unused dependency. I will remove it.